variables.ini.dist

```diff
@@ -84,6 +84,7 @@ TEST_RUNNER_PASS = ; test_secret
 MOCK_USER = ; test_user
 USE_MOCK_AUTH = ; False
 #local.py - site configuration
+TIME_ZONE = ; America/Phoenix
```

Comment:
I'm not sure how this all works when you consider that you can set `TIME_ZONE` on Celery, 

```
# General Celery Settings
CELERY_ENABLE_UTC = True
CELERYD_PREFETCH_MULTIPLIER = 1
CELERY_TIMEZONE = TIME_ZONE
```
source: [atmosphere/settings/__init__.py](https://github.com/iPlantCollaborativeOpenSource/atmosphere/blob/master/atmosphere/settings/__init__.py#L354-L356)

... but I assume without researching in Celery docs, this is all A - :ok: